config/environments/production.rb

```diff
@@ -28,7 +28,7 @@
   config.require_master_key = !IS_DOCKER_BUILD
 
   # Disable serving static files from `public/`, relying on NGINX/Apache to do so instead.
-  config.public_file_server.enabled = true
+  config.public_file_server.enabled = false
```

Comment:
This says "don't use Rails to serve `public/` files" (since NGINX will do it, instead).